ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -741,7 +757,64 @@
           - email:
               notify-every-unstable-build: true
               recipients: projectantrea-dev@googlegroups.com
-          wrappers: []
+          wrappers:
+          - credentials-binding:
+            - text:
+                credential-id: WORKFORCE_POOL # Jenkins secret that stores the cloud resource pool id
+                variable: WORKFORCE_POOL
+      - 'cloud-{name}-{test_name}-credential':
+          test_name: gke
+          node: antrea-credential
+          description: 'This is the cloud credential job for antrea gke tests.'
+          builders:
+          - shell: |-
+              #!/bin/bash
+              # Prevent exposing credentials in the console output by adding set +x
+              # This is to avoid other developers removing this set+x by accident
+              set +x
+              token_file="/var/lib/jenkins/ci_properties.txt"
+              if [ -e "$token_file" ]; then
+                  modification_timestamp=$(stat -c %Y "$token_file")
+                  current_timestamp=$(date +%s)
+                  time_difference=$((current_timestamp - modification_timestamp))
+              else
+                  time_difference=36001
+              fi
+              # The credentials are valid for 12 hours, to allow sufficient time for job execution, the reuse threshold is set to 10 hours.
```

Comment:
the credentials are valid for 12 hours but the refresh job executes every 2 days? That doesn't sound right. As @tnqn pointed out, there should always be up-to-date credentials, so we can run jobs manually (for debugging, or before a release)

it feels like if the credentials are valid for 12 hours, we should run the refresh job every 6 hours to be on the safe side (or maybe 9). But we should only run the conformance job once every 2 days.